ocamlformat-rpc.opam

```diff
@@ -10,9 +10,24 @@ bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
 depends: [
   "dune" {>= "2.8"}
   "ocaml" {>= "4.08" & < "4.15"}
+  "ocamlformat-rpc-lib"
   "alcotest" {with-test}
-  "ocamlformat" {= version}
```

Comment:
The dune file still lists `ocamlformat` as a library dependency. Where would that dependency come from if this package is removed?

Reply:
That's because #2022 renamed ocamlformat_lib to ocamlformat, but this dependency was always here.